src/cf-layout/src/cf-layout.less

```diff
@@ -665,18 +665,22 @@
     &_heading {
         .heading-1();
 
-        .respond-to-max(800px, {
+        .respond-to-max(@bp-sm-max, {
             .heading-2();
         });
     }
 
-    &_subhead {
+    &_subhead,
+    &_cta {
```

Comment:
Can't use `.lead-paragraph`, because https://github.com/cfpb/capital-framework/pull/269/files#diff-5140a128023e2d3dcfd06c7e5b80e2c5R681

Good point about the button-style CTA. Maybe fix by changing this selector to `p&_cta?` I'm not certain what the intended markup for using the button-style CTA is, though?